var/spack/repos/builtin/packages/julia/package.py

```diff
@@ -20,11 +20,11 @@ class Julia(Package):
     # depends_on("pkg-config", deptypes='build')
 
     # Combined build-time and run-time dependencies:
-    depends_on("binutils", deptypes='build')
-    depends_on("cmake @2.8:", deptypes='build')
-    depends_on("git")
-    depends_on("openssl")
-    depends_on("python @2.7:2.999")
+    depends_on("binutils", deptypes=('build', 'run'))
+    depends_on("cmake @2.8:", deptypes=('build', 'run'))
+    depends_on("git", deptypes=('build', 'run'))
+    depends_on("openssl", deptypes=('build', 'run'))
+    depends_on("python @2.7:2.999", deptypes=('build', 'run'))
```

Comment:
This looks good.

There are two more dependency declarations further down; these are purely run-time dependencies.